### PR TITLE
Feature/backport coverage

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

Backport all the changes for *san and coverage handling 

- [x] :hankey: Bugfix

### CODE REVIEW

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/job/arangodb-ANY-coverage/201/

File: js/.eslintignore [hunk 1]
```diff
@@ -1 +1,2 @@
-apps/system/_admin/aardvark/APP/react/build/*
\ No newline at end of file
+apps/system/_admin/aardvark/APP/react/build/*
+client/modules/@arangodb/testutils/instance.js
```
Comment (KVS85) [inline js/.eslintignore:2]: Why?

**Comment by KVS85** [comment]
- [x] https://jenkins.arangodb.biz/job/arangodb-ANY-coverage/203/

File: js/.eslintignore (hunk 1 above)
Comment (dothebart) [reply js/.eslintignore:2]: our linter is older and doesn't understand the #pid syntax in that file that @mpoeter introduced. However, our V8 does.

**Review by naushniki — APPROVED** [review]

**Pull request merged**